activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -178,6 +195,14 @@ def disable_referential_integrity #:nodoc:
         end
       end
 
+      # CONNECTION MANAGEMENT ====================================
+
+      # Clears the prepared statements cache.
+      def clear_cache!
+        reload_type_map
```

Comment:
PG and SQLite3 adapters does not call `super`.
https://github.com/rails/rails/blob/v5.0.0.beta3/activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb#L230-L232
https://github.com/rails/rails/blob/v5.0.0.beta3/activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb#L149-L151

https://github.com/rails/rails/blob/v5.0.0.beta3/activerecord/lib/active_record/connection_adapters/abstract_adapter.rb#L356-L358